tests/test_irbempy.py

```diff
@@ -149,6 +149,36 @@ def test_get_Lstar_OPQuiet(self):
         for key in expected.keys():
             numpy.testing.assert_almost_equal(expected[key], actual[key], decimal=6)
 
+    def test_get_Lstar_OPQuiet_multi(self):
+        """Test Lstar on OPQ forcing multiprocess"""
+        cpu_actual = spacepy.config['ncpus']
+        # To trigger a worker pool, number of calcs must be
+        # more than double number of cpus
+        spacepy.config['ncpus'] = 4
```

Comment:
If this test errors, it'll leave ncpus set to the new value, which might affect other tests in the test suite. This can be addressed by wrapping everything else in a try/finally.

Reply:
Good catch, I'll wrap it.